hardcode version of hwinfo, this must be changed in sync 
with hwinfo version changes when updating the package
--- a/Makefile
+++ b/Makefile
@@ -12,12 +12,8 @@ export SO_LIBS
 
 GIT2LOG := $(shell if [ -x ./git2log ] ; then echo ./git2log --update ; else echo true ; fi)
 GITDEPS := $(shell [ -d .git ] && echo .git/HEAD .git/refs/heads .git/refs/tags)
-BRANCH  := $(shell [ -d .git ] && git branch | perl -ne 'print $$_ if s/^\*\s*//')
-ifdef HWINFO_VERSION
-VERSION := $(shell echo ${HWINFO_VERSION} > VERSION; cat VERSION)
-else
-VERSION := $(shell $(GIT2LOG) --version VERSION ; cat VERSION)
-endif
+BRANCH  := master
+VERSION := $(shell echo 21.71 > VERSION; cat VERSION)
 PREFIX  := hwinfo-$(VERSION)
 
 include Makefile.common
